sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -322,6 +322,56 @@ def run_job(self):
         os.chdir(current_path)
 
 
+class UpdateDockerData(AbstractJob):
+    '''
+    Performs the commands 'docker info' & 'docker images' and saves the 
+    results to a json file where the PHP user can read the data, used for
+    the submitty docker interface
+    '''
+    def run_job(self):
+
+        tgt_path = Path(DATA_DIR, 'docker_data', 'submitty_docker.json')
+        log_path = Path(DATA_DIR, 'logs', 'docker_interface_logs')
+
+        today = datetime.datetime.now()
+        log_file_path = os.path.join(log_path, 
+                                "{:04d}{:02d}{:02d}.txt".format(today.year, today.month,
+                                today.day))
+        full_time = today.strftime("%d/%m/%Y %H:%M:%S")
+
+        try:
+            docker_info = subprocess.run(["docker", "info", "--format", "{{json .}}"], stdout=subprocess.PIPE)
+            docker_images = subprocess.run(["docker", "images", "--format", "{{json .}}"], stdout=subprocess.PIPE)
+
```

Comment:
Use the [docker python sdk](https://docker-py.readthedocs.io/en/stable/) and not subprocess.